src/types/banner.ts

```diff
@@ -0,0 +1,9 @@
+export interface Banner {
+  title?: string;
+  content?: string;
+  isActionButton?: boolean;
+  buttonUrl?: string;
+  buttonDisplayText?: string;
+  isFullScreen?: boolean;
+  hasCloseButton?: boolean;
+}
```

Comment:
Are none of these fields required?

Reply:
yep, content is required, change in 771b70a